src/routes/Agreements/AgreementsView.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import { JCheckbox, JText, JRaisedButton } from 'components/base'
+import { setAgreements, getAgreements, checkAgreements } from 'utils/agreements'
+
+type StateProps = {|
+  +isDisabled: boolean,
+|}
+
+const conditionsArray = [
```

Comment:
It would be better to assign texts as key-value pairs. So when we will change something, we do not need to support the same order in multiple places.

Reply:
Why? Because we will check and modify them separately. So we could discard `JSON.parse` which is a little too resource-consuming.